app/src/main/java/beepbeep/pixelsforredditx/common/SnackbarOnlyOne.kt

```diff
@@ -24,11 +27,19 @@ class SnackbarOnlyOne() {
                     }
                 })
                 snackbar = this
+                val backgroundColor = getThemeBackgroundColor(this@apply.view.context)
```

Comment:
Why sudden we need this in this PR bro?

Reply:
ah this is to fix another bug where the text is not visible in Snackbar in dark theme due to wrong color